modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -98,7 +98,12 @@
 				applied_style = STYLE_VOX
 			else if((DIGITIGRADE in dna.species.species_traits) && (wear_suit.mutant_variants & STYLE_DIGITIGRADE))
 				applied_style = STYLE_DIGITIGRADE
-
+		if(applied_style == STYLE_TAUR_SNAKE) //i hate this but i don't know how else to do it
+			RemoveElement(/datum/element/footstep)
+			AddElement(/datum/element/footstep, FOOTSTEP_MOB_SNAKE, 0.6, -6)
+		else
+			RemoveElement(/datum/element/footstep)
+			AddElement(/datum/element/footstep, FOOTSTEP_MOB_HUMAN, 0.6, -6)
```

Comment:
this is giving me migraines

Reply:
Shut up chungus, you have the worst smelling nut cheese fungus amongus.